example/e2e-example.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
```

Comment:
I think this file shouldn't belong to the example, the example is mainly for demo purposes and this script won't demo things :p I would move this script back to e2e folder and keep a nice and eloquent readme here.